subprojects/core/src/main/java/org/gradle/api/internal/file/copy/FileCopyDetailsInternal.java

```diff
@@ -23,4 +23,8 @@ public interface FileCopyDetailsInternal extends FileCopyDetails {
     boolean isDefaultDuplicatesStrategy();
 
     CopySpecResolver getSpecResolver();
+
+    String getDisplayName();
```

Comment:
This is already implemented by all implementors, but wasn't present on the interface.

Reply:
💭 would it make sense to make this implement `Describable` instead?

Reply:
Looks like it would.